### PR TITLE
start of some clang-tidy fixes

### PR DESCRIPTION
**Pull request by zingale**
mostly updates header guards to conform to our standards and not conflict with reserved patterns

### CODE REVIEW

**Comment by zingale** [comment]
requires #1142

File: unit_test/test_jac/jac_zones.H [hunk 1]
```diff
@@ -1,5 +1,5 @@
-#ifndef _react_zones_H_
-#define _react_zones_H_
+#ifndef REACT_ZONES_H
```
Comment (yut23) [inline unit_test/test_jac/jac_zones.H:1]: update to `JAC_ZONES_H` while we're here?

File: unit_test/test_rhs/rhs_zones.H [hunk 2]
```diff
@@ -1,5 +1,5 @@
-#ifndef _react_zones_H_
-#define _react_zones_H_
+#ifndef REACT_ZONES_H
```
Comment (yut23) [inline unit_test/test_rhs/rhs_zones.H:1]: same here

**Review by yut23 — APPROVED** [review]

File: unit_test/test_rhs/rhs_zones.H (hunk 2 above)
Comment (zingale) [reply unit_test/test_rhs/rhs_zones.H]: done

**Pull request merged**